apps/web/vitest.setup.ts

```
@@ -72,222 +72,50 @@ const mockSearchResults = [
-      }
-
-      // Category browsing
-      return {
-        emojis: args?.page === 0 ? mockCategoryEmojis : [],
-        hasMore: args?.page === 0,
-        page: args.page || 0,
-        pageSize: args.pageSize || 200,
-        totalCount: mockCategoryEmojis.length,
-      };
+  }
+  
```

ESLint

[error] Delete `··`

(prettier/prettier)